.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: go
-
+services:
+  - docker
 # Needed for e2e tests
 sudo: true
-go: 1.9.x
+go: 1.x
```

Comment:
I am assuming there's a good reason for being less strict with go version here.

Reply:
It is better than updating travis every time a new version comes out.